springdb/frontend/package.json

```diff
@@ -42,6 +42,7 @@
     "@mui/lab": "^5.0.0-alpha.69",
     "@mui/material": "^5.4.2",
     "@testing-library/jest-dom": "^5.16.2",
+    "cors": "^2.8.5",
```

Comment:
Is this used anywhere?  Should it be removed?

Reply:
I don't use it. Probably should remove it. @Yan,  please let me know if we should delete this?

Reply:
I'm not seeing this in my package.json. If you're not using it, let's delete